src/components/tooltip/Marker.js

```diff
@@ -173,36 +173,47 @@ class MarkerTooltipContents extends React.PureComponent<Props> {
    * they have threadId field.
    */
   _renderThreadDetails(): TooltipDetailComponent[] {
-    const { marker, threadName, threadIdToNameMap } = this.props;
+    const { marker, threadIdToNameMap } = this.props;
     const data = marker.data;
 
-    if (!data || data.threadId === undefined) {
```

Comment:
note: I inverted this if condition, I thought this would be easier to follow in this direction.